modules/client-configuration/pages/clients-almalinux.adoc

```diff
@@ -33,9 +33,13 @@ You do not need to disable SELinux to register {almalinux} clients to {productna
 
 Before you can register {almalinux} clients to your {productname} Server, you need to add the required software channels, and synchronize them.
 
+To check whether the product and architecture you need is supported, see xref:client-configuration:supported-features.adoc[].
+
+
 ifeval::[{suma-content} == true]
 
-The products you need for this procedure are:
+For example, when working with x86_64 architecture, you need this product:
```

Comment:
I see a problem with this approach: for the channels, the architecture is `x86_64`, but at the matrix you render it as `x86-64`. For Z series, the architecture for the channel is ' s390x` but at the matrix you have "IBM Z". For ARM64, the architecture is `aarch64` for the channels, but at the matrix you have `ARM`.

![image](https://user-images.githubusercontent.com/4226070/135072072-440f50e1-f4fd-4cee-876a-3074c7fd9ab5.png)


So this won't work unless the users can copy the architecture from the matrix.

Reply:
This is correct., but I prefer to separate this kind of tidy-up in separate task as it is not regression.

Reply:
Agreed if this falls in line with former process than merge as is. We can clean up in a new PR.

Reply:
> This is correct.
However, the changes added are in line with how the affected docuents were updated in the past, and the existing content is not regression.
> I would therefore recommend that we to separate the kind of tidy-up that would definitely be good in separate PR.

I agree on other situations, but not for this one. What will happen when a customer tries to add `almalinux8-ARM` via `mgr-sync`?

If you want cleanup, then please change this message to:

```
Two architectures are supported: `x86_64` and `aarch64`

For example, when working with `x86_64` architecture, you need this product:
```
Even without using entitires, if needed.

FMPOV this is easy to do, and make things understandable while we get the problem fixed with a tidy up. Specially because I don't think the tidy up is going to happen in the next few days, in the for 4.2.3 (otherwise feel free to ignore this comment).

Remember to check if all OS form the card are documented at the table, I saw AlmaLinux, but could be others.

Reply:
@juliogonzalez changed for the clients from this round of changes.